docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+If you would like to contribute new example apps to the `elasticsearch-labs` repo, we would love to hear from you!
+
+## Before you start
+
+Prior to opening a pull request, please:
+- Create an issue to [discuss the scope of your proposal](https://github.com/elastic/elasticsearch-labs/issues). We are happy to provide guidance to make for a pleasant contribution experience.
+- Sign the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/). We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
+
+## Contributing to Python notebooks 📒
+
+### Why
+
+* The main goal of this repo is to help people learn about solving various problems with the Elastic Stack using step-by-step interactive guides and specific applications.
+* Remember your target audience: developers who want to try out some technology with Elastic. They may not be familiar with all the technologies.
+
+### Where
+
+* Select a folder under [notebooks](../notebooks/README.md) that matches the category of your notebook. If none of them match, create a new folder.
+* Create a folder under the category for your notebook.
```

Comment:
I wonder if creating a separate folder for every notebook isn't overkill? I'm leaning towards updating the README in the notebook category folder to summarize what each notebook does. WDYT?

Reply:
Good point, I removed that bulletpoint and added one about updating the `README.md` file.